docs/reference/changelog.rst

```diff
@@ -4,6 +4,14 @@
 Changelog
 *********
 
+4.9.1 (2025-Feb-12)
+-------------------
+
+Application
+===========
+
+- Load python plugins after the emitter has been initialized so they can be logged.
+
```

Comment:
This will need the usual [link to the GH release](https://github.com/canonical/craft-application/blob/78be7a9f50cebe94deb09108f396bac441fa13d5/docs/reference/changelog.rst?plain=1#L29).